### DISCUSSION TITLE
Interface - Max lenght

### DISCUSSION THREAD

**Discussion by akashha** [netbox-community/netbox#6245] [Ideas]
Hello,

You can't create interface description more than 100 characters on a device (/dcim/interfaces/add/?device=24).
If you edit the interface, you can set 200 characters in description (/dcim/interfaces/24/edit/).

Can you set the max directly to 200 in create interface description.
Sorry for my english, i am french.

When create :
`<input type="text" name="description" maxlength="100" class="form-control" placeholder="None" id="id_description">`
When edit :
`<input type="text" name="description" maxlength="200" class="form-control" placeholder="None" id="id_description">`

Cordialy.

**Comment by candlerb** [comment]
Netbox version?

EDIT: I can confirm this is the case with Netbox v2.11.1. I think you can raise a bug issue for this.

**Reply by akashha** [reply]
thx. v2.11.0, i update in v2.11.1

**Reply by candlerb** [reply]
I can't work out how the code handles this, and I can see other anomolies. E.g. the "Label" field has no maxlength when creating an interface:

```
<input type="text" name="label_pattern" class="form-control" placeholder="Label" id="id_label_pattern">
```

But it does when editing an interface:

```
<input type="text" name="label" maxlength="64" class="form-control" placeholder="Label" id="id_label">
```

**Reply by akashha** [reply]
Yes, i have update and i confirme the issue.

**Reply by akashha** [reply]
Issue in progress. Thx.
